src/cf-vars.less

```diff
@@ -72,33 +72,33 @@
 // Color variables
```

Comment:
Do we necessarily need them? Are we going to update these if 18F changes any of them in the future? Or is this just a starting point and we'll go on our own from here?